### PR TITLE
Migrate to Gradle Consistent Versions

### PR DESCRIPTION
**Pull request by dansanduleac**

### CODE REVIEW

**Review by draganrelu — DISMISSED** [review]
LGTM

**Comment by draganrelu** [comment]
@ashrayjain can we get a +1 on this? this is blocking the #54 throughout repos

**Review by ashrayjain — DISMISSED** [review]

File: versions.props [hunk 1]
```diff
@@ -3,9 +3,4 @@ junit:junit = 4.12
 one.util:streamex = 0.6.8
 org.assertj:assertj-core = 3.12.1
 org.mockito:mockito-core = 2.24.5
-
-# Conflicts
-com.google.code.findbugs:jsr305 = 3.0.2
-com.google.errorprone:error_prone_annotations = 2.3.3
-org.checkerframework:checker-qual = 2.5.3
 org.objenesis:objenesis = 3.0.1
```
Comment (SerialVelocity) [inline versions.props]: Can this be removed as well? We don't explicitly depend on this.

Comment (dansanduleac) [reply versions.props]: when i removed it, the transitive constraints bumped it donw to 2.6, and i wasn't sure if that's safe

**Review by draganrelu — APPROVED** [review]

**Review by SerialVelocity — APPROVED** [review]

**Pull request merged**